content/docs/introduction/faq.md

```diff
@@ -47,6 +63,11 @@ file in each repository.
 Prometheus is released under the
 [Apache 2.0](https://github.com/prometheus/prometheus/blob/master/LICENSE) license.
 
+### What is the plural of Prometheus?
+
+After extensive research it has been determined that the correct plural of
+'Prometheus' is 'Prometheis'.
```

Comment:
I thought the conclusion was more nuanced/ambivalent, and it depends on assumptions and preferences. I'd leave this section out for now. If we want to write it properly, we need @beorn7  :)